src/tests/unit/tests/reports/components/assessment-report-step-header.test.tsx

```diff
@@ -91,27 +105,30 @@ describe('AssessmentReportStepHeader', () => {
                                 />
                             );
 
-                            const wrapper = shallow(component);
-
                             it(`will have OutcomeChip with count of ${expectedCount} and outcomeType of ${outcomeType}`, () => {
-                                const chip = wrapper.find(OutcomeChip).getElement();
-                                expect(chip).toEqual(
-                                    <OutcomeChip count={expectedCount} outcomeType={outcomeType} />,
-                                );
+                                render(component);
+                                const chip = getMockComponentClassPropsForCall(OutcomeChip);
+                                expect(chip.count).toEqual(expectedCount);
+                                expect(chip.outcomeType).toEqual(outcomeType);
```

Comment:
Changes done as suggested

Reply:
Here we need to keep the render(component) inside the test case to have values in the Snapshot. If we removed the render(component), all the 18 snapshots give undefined value instead of props.